frontend/src/components/BookPage.js

```diff
@@ -460,49 +525,31 @@ class BookPage extends Component {
               </Select>
             </FormControl>
           </Grid>
-        { this.props.bookNotFound ? "" :
-          <Grid item xs={12} align="center">
-            <Typography component="h5" variant="h5">
-               {this.props.type == 0 ?
-                t("You are SELLING BTC for {{currencyCode}}",{currencyCode:this.props.bookCurrencyCode})
-               :
-                (this.props.type == 1 ?
-                  t("You are BUYING BTC for {{currencyCode}}",{currencyCode:this.props.bookCurrencyCode})
-                :
-                  t("You are looking at all")
-                )
-               }
-            </Typography>
-          </Grid>
+          { this.props.bookNotFound ? <></> :
+            <Grid item xs={12} align="center">
+              <Typography component="h5" variant="h5">
+                {this.getTitle()}
+              </Typography>
+            </Grid>
           }
-
-        { this.props.bookNotFound ?
-          this.NoOrdersFound()
-          :
           <Grid item xs={12} align="center">
-            {/* Desktop Book */}
-            <MediaQuery minWidth={930}>
-              <Paper elevation={0} style={{width: 925, maxHeight: 500, overflow: 'auto'}}>
-                  {this.bookListTableDesktop()}
-              </Paper>
-            </MediaQuery>
-
-            {/* Smartphone Book */}
-            <MediaQuery maxWidth={929}>
-              <Paper elevation={0} style={{width: 395, maxHeight: 450, overflow: 'auto'}}>
-                  {this.bookListTablePhone()}
-              </Paper>
-            </MediaQuery>
-           </Grid>
-          }
+            {this.mainView()}
+          </Grid>
           <Grid item xs={12} align="center">
-            { !this.props.bookNotFound ?
-            <Button variant="contained" color='primary' to='/make/' component={Link}>{t("Make Order")}</Button>
-            : null
-            }
-            <Button color="secondary" variant="contained" to="/" component={Link}>
-                {t("Back")}
-            </Button>
+            <ButtonGroup variant="contained" aria-label="outlined primary button group">
+              { !this.props.bookNotFound ?
+                <>
+                  <Button variant="contained" color='primary' to='/make/' component={Link}>{t("Make Order")}</Button>
+                  <Button color='inherit' style={{color: '#111111'}} onClick={this.handleClickView}>
+                    { this.state.view == 'depth' ? t("List") : t("Depth") }
```

Comment:
What do you think of using "Chart" instead of "Depth"? 
We could also add a small icon on this button so it is highlighted. For example:
```javascript
// Chart Button
import BarChartIcon from '@mui/icons-material/BarChart';
// List Button
import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted';
```